renku/core/models/projects.py

```diff
@@ -220,14 +203,32 @@ class Meta:
 
         rdf_type = [schema.Project, prov.Location]
         model = Project
+        unknown = EXCLUDE
+
+    name = fields.String(schema.name, missing=None)
+    created = fields.DateTime(schema.dateCreated, missing=None)
+    updated = fields.DateTime(schema.dateUpdated, missing=None)
+    version = fields.String(schema.schemaVersion, missing=1)
+    creator = fields.Nested(schema.creator, PersonSchema, missing=None)
+    _id = fields.Id(init_name='id', missing=None)
+
+    @pre_load
+    def translate_old_project(self, data, **kwargs):
```

Comment:
Same as above, can we use a schema inside a migration to do this or would this break things?

Reply:
This is not required by the migrations themselves but by the code that check the project version (which infinitely tries to create project causing an SO). I've changed that code and removed this from here.